src/hibernate-resume/hibernate-resume-config.h

```diff
@@ -16,6 +16,8 @@ typedef struct EFIHibernateLocation {
         sd_id128_t uuid;
         uint64_t offset;
 
+        bool auto_swap;
```

Comment:
i think the other fields are mostly self-explanatory to some degree. can you add a brief comment here explaining what this is about?